### PR TITLE
Adding Daily percentage classes

### PR DESCRIPTION
**Pull request by dgingi**

### CODE REVIEW

**Comment by dgingi** [comment]
@OryJonay Please take a look at my last 2 commits and approve the changes

File: oddsgym/envs/base.py [hunk 1]
```diff
@@ -137,7 +136,29 @@ def step(self, action):
...
+            A list of games, with their betting odds.
+        results : array of shape (1, n_odds)
+
+        Returns
+        -------
+        reward : float
+            The amount of reward returned after previous action
+        """
+        used_results = numpy.ones_like(results)
+        zero_rows_count = numpy.sum(~results.any(1))
```
Comment (OryJonay) [inline oddsgym/envs/base.py]: Nice :-)

File: oddsgym/envs/daily_bets.py [hunk 2]
```diff
@@ -196,7 +195,7 @@ def __init__(self, odds, odds_column_names, results=None):
         self.action_space = gym.spaces.Box(low=numpy.array([[self.action_space.low[0]] + [0.] * self._odds.shape[1]
                                                             for i in numpy.arange(self.action_space.shape[0])]),
                                            high=numpy.array([[self.action_space.high[0]] + [1.] * self._odds.shape[1]
-                                                             for i in numpy.arange(self.action_space.shape[0])]))
+                                                            for i in numpy.arange(self.action_space.shape[0])]))
```
Comment (OryJonay) [inline oddsgym/envs/daily_bets.py]: The "for" needs to be aligned with the inner brackets (just FYI)

**Review by OryJonay — CHANGES_REQUESTED** [review]
I'll fix the comment about the aligning the "for", add ThreeWaySoccerPercentageEnv and do another go on the docs.

File: tests/test_three_way_daily_percentage_env.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                                                    (numpy.array([[3] + [0.1] * 3] * 2), -2),
+                                                    (numpy.array([[4] + [0.1] * 3] * 2), 2),
+                                                    (numpy.array([[5] + [0.1] * 3] * 2), 0),
+                                                    (numpy.array([[6] + [0.1] * 3] * 2), -2),
+                                                    (numpy.array([[7] + [0.1] * 3] * 2), 0)])
+def test_step(three_way_daily_percentage_env, action, expected_reward):
+    odds, reward, done, _ = three_way_daily_percentage_env.step(action)
+    assert reward == expected_reward
+    assert not done
+    assert three_way_daily_percentage_env.current_step == 1
```
Comment (OryJonay) [inline tests/test_three_way_daily_percentage_env.py]: Add more cases

File: tests/test_three_way_daily_env.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+                                                                            ['PSG', 'FCB'], ['MTA', 'MCB']]))
+
+
+@pytest.mark.parametrize("action,expected_reward", [((0, 0), 0), ((1, 1), 2), ((2, 2), 0), ((3, 3), -2), ((4, 4), 2),
+                                                    ((5, 5), 0), ((6, 6), -2), ((7, 7), 0)])
+def test_step(three_way_daily_env, action, expected_reward):
+    odds, reward, done, _ = three_way_daily_env.step(action)
+    assert reward == expected_reward
+    assert not done
+    assert three_way_daily_env.current_step == 1
```
Comment (OryJonay) [inline tests/test_three_way_daily_env.py:25]: Add more cases

**Review by OryJonay — CHANGES_REQUESTED** [review]
@dgingi Please review my latest commit and add more cases to the "test_step" of the new soccer environments.

**Pull request merged**